_maps/shuttles/shiptest/nanotrasen_skipper.dmm

```diff
@@ -1,5291 +1,7101 @@
+	pixel_x = -8
 	},
-/turf/open/floor/noslip,
-/area/ship/engineering)
-"EN" = (
-/obj/structure/cable{
-	icon_state = "4-8"
+/turf/open/floor/plasteel/dark,
+/area/ship/crew/canteen)
+"vH" = (
+/obj/machinery/ore_silo,
+/obj/structure/window/reinforced{
+	dir = 8
 	},
-/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden/layer4{
+/obj/item/multitool,
+/obj/effect/turf_decal/corner/opaque/black/three_quarters{
 	dir = 4
 	},
-/obj/machinery/atmospherics/pipe/simple/supply/hidden/layer2{
-	dir = 4
+/obj/effect/turf_decal/industrial/warning{
+	dir = 8
 	},
-/turf/open/floor/plasteel/grimy,
-/area/ship/crew/solgov)
-"EW" = (
-/obj/structure/cable{
-	icon_state = "1-2"
+/turf/open/floor/plasteel,
+/area/ship/cargo/office)
+"vO" = (
+/obj/effect/turf_decal/corner/opaque/blue/half,
+/turf/open/floor/plasteel/tech/techmaint,
+/area/ship/engineering/atmospherics)
+"vP" = (
+/obj/structure/sign/warning/nosmoking{
+	pixel_y = 36
 	},
-/obj/structure/cable{
-	icon_state = "2-8"
+/obj/structure/sink{
+	pixel_y = 22
 	},
-/obj/machinery/atmospherics/pipe/simple/supply/hidden/layer2,
-/turf/open/floor/plating,
-/area/ship/engineering)
-"EZ" = (
-/obj/machinery/atmospherics/components/unary/vent_pump/on/layer2{
+/obj/effect/turf_decal/corner/opaque/blue/mono,
+/obj/machinery/atmospherics/pipe/manifold/scrubbers/hidden/layer4{
 	dir = 1
 	},
-/obj/machinery/power/apc/auto_name/south,
-/obj/structure/cable,
-/turf/open/floor/plasteel/grimy,
-/area/ship/crew/solgov)
-"Fa" = (
-/obj/structure/table,
-/obj/item/reagent_containers/food/drinks/beer{
-	pixel_x = 6;
-	pixel_y = 2
+/obj/machinery/atmospherics/pipe/simple/supply/hidden/layer2{
+	dir = 10
 	},
-/obj/item/reagent_containers/food/drinks/beer{
-	pixel_x = -7;
-	pixel_y = 6
+/obj/structure/cable{
+	icon_state = "4-8"
 	},
+/turf/open/floor/plasteel/mono/white,
+/area/ship/medical)
+"vR" = (
 /obj/effect/turf_decal/corner/opaque/red/diagonal,
 /obj/effect/turf_decal/corner/opaque/yellow/diagonal{
 	dir = 4
 	},
+/obj/structure/chair{
+	dir = 4
+	},
 /turf/open/floor/plasteel,
 /area/ship/crew/canteen)
-"Fg" = (
-/obj/structure/sign/departments/medbay/alt{
-	pixel_x = 32
+"vW" = (
+/obj/machinery/atmospherics/pipe/manifold/scrubbers/visible/layer4{
+	dir = 4
+	},
+/obj/machinery/atmospherics/pipe/simple/supply/hidden/layer2{
+	dir = 4
 	},
-/turf/open/floor/carpet/nanoweave/blue,
-/area/ship/medical)
-"Fo" = (
 /obj/structure/cable{
-	icon_state = "4-8"
+	icon_state = "2-8"
 	},
-/obj/machinery/firealarm{
-	dir = 1;
-	pixel_y = 28
+/obj/structure/cable{
+	icon_state = "1-2"
 	},
-/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden/layer4{
-	dir = 4
+/obj/structure/catwalk/over,
+/turf/open/floor/plating,
+/area/ship/engineering/atmospherics)
+"vY" = (
+/obj/machinery/atmospherics/pipe/simple/green/visible{
+	dir = 10
 	},
-/obj/machinery/atmospherics/pipe/simple/supply/hidden/layer2{
+/obj/structure/fireaxecabinet{
+	dir = 8;
+	pixel_x = 32
+	},
+/obj/machinery/suit_storage_unit/inherit/industrial,
```

Comment:
This SSU appears to only be grabbing the firesuit and gas mask, leaving the water tank and helmet out on top. I'm not actually sure why it isn't taking those since it should have the open slots available for them.

Reply:
from what i see it takes the firesuit first and places it into the aux slot, which ????? no fucking clue, im gonna make an industrial SSU subtype for it